controllers/articles/index.js

```diff
@@ -94,7 +149,7 @@ export default {
           body,
           title,
           readTime,
-          categoryId
+          /* categoryId */
```

Comment:
Please could you explain why this was commented out?

Reply:
forgot to uncomment it. would remove it seeing that I put it again in the object.